src/main/scala/com/gd/twitterstreamingtToDatalake/StreamingTweetsJob.scala

```diff
@@ -0,0 +1,38 @@
+package com.gd.twitterstreamingtToDatalake
+
+import org.apache.log4j.{Level, Logger}
+
+
+object StreamingTweetsJob extends SparkSessionWrapper {
+
+  //Set the Logger Level
+
+  Logger.getLogger("org").setLevel(Level.ERROR)
```

Comment:
I suppose you've set up the `ERROR` level for debugging purposes.


In general / JFYI,
Configuration of Logging should be externalized to a config file, e.g. https://www.tutorialspoint.com/log4j/log4j_configuration.htm

Reply:
Fixed.